internal/scorecard/alpha/testdata/bundle/manifests/memcached-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -14,7 +14,7 @@ metadata:
             "size": 3
           },
           "status": {
-            "nodes": null
+            "status": "Reconciled"
```

Comment:
I am not sure, but I do not think that the csv are gen now with the tag in the name as`memcached-operator.v0.0.1.clusterserviceversion.yaml`. I think we need to verify that and update the mock testadata to reflect the current behaviour.

Reply:
Also, it may be out of scope of this PR.